src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,12 @@
+import { Button } from 'antd'
+import classNames from 'classnames'
+
+import { IIconButtonProps } from './IconButton-types'
+
+import './IconButton-styles.less'
+
+export function IconButton(props: IIconButtonProps) {
+	const { className } = props
+
+	return <Button {...props} className={classNames('icon-button', className)} />
+}
```

Comment:
De cette façon, `className` n'existe plus dans `restProps`.
```suggestion
export function IconButton(props: IIconButtonProps) {
	const { className, ...restProps } = props

	return <Button {...restProps} className={classNames('icon-button', className)} />
}
```